[2022-07-23 09:13:52,236] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.upload_to_s3_2019-01 manual__2022-07-23T09:13:39.645249+00:00 [queued]>
[2022-07-23 09:13:52,253] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.upload_to_s3_2019-01 manual__2022-07-23T09:13:39.645249+00:00 [queued]>
[2022-07-23 09:13:52,256] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:13:52,257] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-23 09:13:52,257] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:13:52,293] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2019-01> on 2022-07-23 09:13:39.645249+00:00
[2022-07-23 09:13:52,304] {standard_task_runner.py:52} INFO - Started process 479 to run task
[2022-07-23 09:13:52,315] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_snowflake', 'upload_to_s3_2019-01', 'manual__2022-07-23T09:13:39.645249+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/localtosnowflake.py', '--cfg-path', '/tmp/tmpvj9zb4b6', '--error-file', '/tmp/tmp01j9s4ql']
[2022-07-23 09:13:52,318] {standard_task_runner.py:80} INFO - Job 52: Subtask upload_to_s3_2019-01
[2022-07-23 09:13:52,494] {task_command.py:371} INFO - Running <TaskInstance: local_to_snowflake.upload_to_s3_2019-01 manual__2022-07-23T09:13:39.645249+00:00 [running]> on host cbd64c50b1d6
[2022-07-23 09:13:52,659] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_snowflake
AIRFLOW_CTX_TASK_ID=upload_to_s3_2019-01
AIRFLOW_CTX_EXECUTION_DATE=2022-07-23T09:13:39.645249+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-23T09:13:39.645249+00:00
[2022-07-23 09:13:52,679] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-23 09:13:52,681] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-23 09:13:53,737] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=local_to_snowflake, task_id=upload_to_s3_2019-01, execution_date=20220723T091339, start_date=20220723T091352, end_date=20220723T091353
[2022-07-23 09:13:53,773] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-23 09:13:53,821] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
